examples/CMakeLists.txt

```diff
@@ -18,6 +18,9 @@ file(
   "${PROJECT_EXAMPLE_DIR}/*.cc"
 )
 
+include(turn_on_warnings)
+turn_on_warnings(${VT_TV_LIBRARY})
```

Comment:
I see this is already done in `src/CMakeLists.txt`, so do you want to target the examples here instead?

Reply:
Fixed